elastica/joint.py

```diff
@@ -209,9 +210,15 @@ class FixedJoint(FreeJoint):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float
+            Rotational damping coefficient of the joint.
+        rest_rotation_matrix: np.array
+            Rest 3x3 rotation matrix from system one to system two at the connected elements.
+            Instead of aligning the directors of both systems directly, a desired rest rotational matrix labeled C_12*
+            is enforced.
     """
 
-    def __init__(self, k, nu, kt):
+    def __init__(self, k, nu, kt, nut=0.0, rest_rotation_matrix=np.eye(3)):
```

Comment:
Avoid using numpy array as a default argument.

Reply:
Fixed in https://github.com/GazzolaLab/PyElastica/pull/135/commits/91d18173b492499cd574b0d13ca9b7a94dace42c